open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -1,19 +1,131 @@
 # Editor in Chief Guide
 
-## Editor in Chief Role & Responsibilities
+## Editor in Chief role & responsibilities
```

Comment:
```suggestion
## Editor in Chief (EiC) role & responsibilities
```
Make it super obvious for highly distracted people (como moi) that this is the acronym we will use throughout the page?